src/storage/src/hummock/sstable_store.rs

```diff
@@ -48,6 +51,48 @@ pub struct SstableStore {
     meta_cache: Arc<LruCache<HummockSSTableId, Box<Sstable>>>,
 }
 
+pub struct BlockStream {
```

Comment:
We will improve this to have a queue with tasks running in background to fetch to queue in future optimization.

Reply:
Please move `BlockStream` to `compactor/sstable_store` to replace `SstableBlocks` . Because we would never use a stream iterator for read request in compute-node.